docs/concepts/v2/concepts/rooms/room-codes/room-codes.mdx

```diff
@@ -48,11 +48,26 @@ Room Codes do not have an expiry, thus, you can reuse Room Codes infinitely. Thi
 
 ### Creating Room Codes
 
-There are multiple ways a Room Code can be created :
-    1. [via 100ms dashboard](https://dashboard.100ms.live/dashboard) : For Rooms created on 100ms dashboard, a Room Code gets generated for every Role in the Room by default.
-    2. via API : If you're using [Create Room API](/server-side/v2/api-reference/Rooms/create-via-api) to create Rooms, you need to explicitly create Room Codes for every Role in the Room. Refer [Create Room Codes API](/server-side/v2/api-reference/room-codes/create-room-code-api). This API allows you to create Room Code for
-    	- every Role in the Room at once 
-   	    - a specific Role in the Room 
+<Tabs id="test-code" items={['via Dashboard','Create Room Code API']} />
+
+<Tab id='test-code-0'>
+
+For Rooms created on [100ms dashboard](https://dashboard.100ms.live/dashboard), a Room Code gets generated for every Role in the Room by default.
+
+![RoomCodeFromJoinRoomModal](/public/guides/JoinRoomModal.png)
```

Comment:
this image is broken for me in the preview link: https://100ms-docs-git-roomcodeinauth-100mslive.vercel.app/docs/concepts/v2/concepts/rooms/room-codes/room-codes

Reply:
```suggestion
![RoomCodeFromJoinRoomModal](/docs/v2/JoinRoomModal.png)
```